simplified-books-database/src/main/java/org/nypl/simplified/books/book_database/BookDatabase.kt

```diff
@@ -42,38 +42,38 @@ class BookDatabase private constructor(
    * A thread-safe map exposing read-only snapshots of database entries.
    */
 
-  private class BookMaps internal constructor() {
+  private class BookMaps() {
 
-    internal val mapsLock: Any = Any()
+    val mapsLock: Any = Any()
```

Comment:
Why is this public now? It's dangerous to expose an internal lock in this way.

Reply:
The modifier was redundant because the whole class is private.

Reply:
Ah, I see!